include/core/bdf.h

```diff
@@ -82,8 +55,9 @@ bdf_coefficients(unsigned int order, const std::vector<double> time_steps);
  * @note At the moment, the highest implemented scheme is @p bdf3 (@p order=3).
  */
 Vector<double>
-bdf_coefficients(const Parameters::SimulationControl::TimeSteppingMethod method,
-                 const std::vector<double> time_steps);
+calculate_bdf_coefficients(
+  const Parameters::SimulationControl::TimeSteppingMethod method,
+  const std::vector<double>                               time_steps);
 
```

Comment:
lines 40-43 could be removed since it is repeated at lines 49-52. I think it is better placed after the `time_steps` parameter